README.md

```diff
@@ -42,3 +42,312 @@ Run linters.
 cargo make clippy
 ```
 
+## Usage
+
+In this section is presented the integration of this library into your application, enabling you to kickstart 
+CQRS/Event sourcing implementation.
```

Comment:
```suggestion
your CQRS/Event sourcing implementation.
```

Reply:
Either `your` or `an` makes the text flow better imo
```suggestion
an CQRS/Event sourcing implementation.
```

Reply:
Maye `a CQRS`?

Reply:
Hm, yeah, I think you're right, but I'm not 100% sure